### ISSUE TITLE
correct fonts.conf for MonoLisa (spacing 100) for kitty terminal - only italic fonts

### ISSUE THREAD

**Issue by Staubgeborener** [MonoLisaFont/feedback#81]
I would like to use the MonoLisa Font with the Kitty Terminal. (Debian 10.2.1).  I can only use some Italic-Font-Versions, but not the bold or regular version. The Kitty FAQ has an interesting post here:
> kitty achieves its stellar performance by caching alpha masks of each rendered character on the GPU, so that every character needs to be rendered only once. This means it is a strictly character cell based display. As such it can use only monospace fonts, since every cell in the grid has to be the same size. Furthermore, it needs fonts to be freely resizable, so it does not support bitmapped fonts.
> 
> If your font is not listed in kitty list-fonts it means that it is not monospace or is a bitmapped font.

```shell
fc-list : family spacing outline scalable | grep -e spacing=100 -e spacing=90 | grep -e outline=True | grep -e scalable=True | grep MonoLisa
MonoLisa,MonoLisa\-MediumItalic:spacing=100:outline=True:scalable=True
MonoLisa,MonoLisa\-BoldItalic:spacing=100:outline=True:scalable=True
MonoLisa,MonoLisa\-LightItalic:spacing=100:outline=True:scalable=True
MonoLisa,MonoLisa\-ExtraLightItalic:spacing=100:outline=True:scalable=True
MonoLisa,MonoLisa\-ThinItalic:spacing=100:outline=True:scalable=True
MonoLisa,MonoLisa\-RegularItalic:spacing=100:outline=True:scalable=True
MonoLisa,MonoLisa\-BlackItalic:spacing=100:outline=True:scalable=True
```

Some MonoLisa fonts are missing, i.e. `MonoLisa-Bold` and `MonoLisa-Regular`. So i created a new `~/.config/fontconfig/fonts.conf`:
```shell
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
<match target="scan">
    <test name="family">
        <string>MonoLisa-Regular</string>
    </test>
    <edit name="spacing">
        <int>100</int>
    </edit>
</match>
</fontconfig>
```

also tried this:

```shell
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
<match target="scan">
    <test name="family">
        <string>MonoLisa</string>
    </test>
    <edit name="spacing">
        <int>100</int>
    </edit>
</match>
</fontconfig>
```

after refreshing everything `fc-cache -frv` still, `kitty list-fonts` shows me only
```shell
MonoLisa
MonoLisa-BlackItalic
MonoLisa-BoldItalic
MonoLisa-ExtraLightItalic
MonoLisa-LightItalic
MonoLisa-MediumItalic
MonoLisa-RegularItalic
MonoLisa-ThinItalic
```
Yes, i restarted my terminal, my wm, logout and login, restarted the whole system, etc.

Same output btw with `fc-list : family spacing outline scalable | grep -e spacing=100 -e spacing=90 | grep -e outline=True | grep -e scalable=True | grep MonoLisa`: Only italic versions.

I managed yesterday with `fonts.conf` that kitty recognized the `MonoLisa-Regular` and `MonoLisa-Bold` fonts, but I had to set up the system again and now it no longer works. I backuped my `fonts.conf`:
```shell
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
<match target="scan">
    <test name="family">
        <string>MonoLisa-Regular</string>
    </test>
    <edit name="spacing">
        <int>100</int>
    </edit>
</match>
<match target="scan">
    <test name="family">
        <string>MonoLisa-Bold</string>
    </test>
    <edit name="spacing">
        <int>100</int>
    </edit>
</match>
</fontconfig>
```
This works perfectly, but not now. Something has changed. So how does the `fonts.conf` have to look correctly with `    <edit name="spacing"><int>100</int></edit>` for it to work?

**Comment by Staubgeborener** [comment]
Ok, my `fonts.conf` example was correct. I had to do `sudo fc-cache -fr` (so i missed `sudo`). There was no error message without `sudo`, so I couldn't figure it out.

**Comment by mpauper** [comment]
Apologies for digging up this old issue, but I see a similar problem and I am so confused. 

I have just bought and downloaded the font. Initially I was trying with OTF but apparently the FAQ and releases pages suggest to stick to TTF. I see the same problem whether I download the version 2.002 or 1.808 of the font, and whether I customize or not the features, but below I will show what happens with version 1.808 and TTF. I am on Pop!_OS 22.04.

When I do `kitty +list-fonts` is see the following for MonoLisa:

```
MonoLisa
    MonoLisa Bold
    MonoLisa Light
    MonoLisa Medium
    MonoLisa Regular
```

So not Italic variants available?

When I do `fc-list : family spacing outline scalable | grep -e spacing=100 -e spacing=90 | grep -e outline=True | grep -e scalable=True | grep "monolisa"`, I see:

```
MonoLisa,MonoLisa Light:spacing=100:outline=True:scalable=True
MonoLisa,MonoLisa Medium:spacing=100:outline=True:scalable=True
MonoLisa:spacing=100:outline=True:scalable=True
```

Again, not all variants are there, and not even the same as the ones with the `kitty +list-fonts` command.

Proof that the font is installed (with the rest of the variants) `fc-list | grep monolisa`:

```
/home/user/.local/share/fonts/Facetype/TrueType/MonoLisa/MonoLisa_Light_Italic.ttf: MonoLisa,MonoLisa Light:style=Light Italic,Italic
/home/user/.local/share/fonts/Facetype/TrueType/MonoLisa/MonoLisa_Light.ttf: MonoLisa,MonoLisa Light:style=Light,Regular
/home/user/.local/share/fonts/Facetype/TrueType/MonoLisa/MonoLisa_Medium.ttf: MonoLisa,MonoLisa Medium:style=Medium,Regular
/home/user/.local/share/fonts/Facetype/TrueType/MonoLisa/MonoLisa_Regular.ttf: MonoLisa:style=Regular
/home/user/.local/share/fonts/Facetype/TrueType/MonoLisa/MonoLisa_Bold.ttf: MonoLisa:style=Bold
/home/user/.local/share/fonts/Facetype/TrueType/MonoLisa/MonoLisa_Bold_Italic.ttf: MonoLisa:style=Bold Italic
/home/user/.local/share/fonts/Facetype/TrueType/MonoLisa/MonoLisa_Medium_Italic.ttf: MonoLisa,MonoLisa Medium:style=Medium Italic,Italic
/home/user/.local/share/fonts/Facetype/TrueType/MonoLisa/MonoLisa_Regular_Italic.ttf: MonoLisa:style=Regular Italic,Italic
```

Please let me know what I am doing wrong, or how do I have to modify the fonts.conf file?
thanks!

**Comment by 3rd** [comment]
@MonoLisaFont two years later this is still an issue, can we get a response on why the italic variants have `spacing=100` and the regular ones do not?

Version 1.808 works correctly, 2.007 does not.
